PaddleAudio/examples/wav2vec2/README.md

```diff
@@ -0,0 +1,37 @@
+# Wav2vec2 models for Speech recognition
```

Comment:
Wav2vec2 Models for Speech Recognition
Title format

Reply:
Maybe Wav2Vec 2.0 would be a precise description.

Reply:
fixed. Will use Wav2vec 2.0 for Speech recognition